src/java.base/share/classes/sun/security/ssl/TrustStoreManager.java

```diff
@@ -384,8 +376,8 @@ private static KeyStore loadKeyStore(
             }
 
             if (!"NONE".equals(descriptor.storeName)) {
-                try (@SuppressWarnings("removal") FileInputStream fis = AccessController.doPrivileged(
-                        new OpenFileInputStreamAction(descriptor.storeFile))) {
+                try (FileInputStream fis =
+                        new FileInputStream(descriptor.storeFile)) {
```

Comment:
wonder if `OpenFileInputStreamAction` could be removed from the codebase ? No other usage detected.. with the exception of test/jdk/sun/security/action/Generify.java which is just a sanity test and could be edited.

Reply:
Yes, I removed it and updated the test. All the action classes will eventually be removed once all dependencies are gone.